xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -340,6 +340,9 @@
     #set($discard = $row.put('doc_author_url', $xwiki.getURL($translatedDoc.author)))
     #set($discard = $row.put('doc_date', $xwiki.formatDate($translatedDoc.date)))
     #set($discard = $row.put('doc_title', $translatedDoc.plainTitle))
+    #if($translatedDoc.getTitle() != $translatedDoc.getPlainTitle())
+	  #set($discard = $row.put('doc_title_computed', true))
```

Comment:
I'm wondering if we should send the raw title back (``doc_title_raw``) instead of the boolean flag (``doc_title_computed``). I would make more sense data-wise but it would also make the JSON a little bigger.

Reply:
Also sending the ``doc_title_raw`` means the logic about if the title is computed moves into the JavaScript code. Maybe it is some old habit of mine, but I prefer to have the logic on the server side code whenever possible. (Probably pointless, as lots of logic end up in JavaScript today anyway.)

Reply:
Ah, ok, now I understand the use case.
The JS code should not get more complicated by this change either, so I will give it a try.